### PR TITLE
Feat: 510 bond modal view

### PR DESCRIPTION
**Pull request by joshuanatanielnm**
#510 

## Description

Update bond modal view based on figma design

## Changes

List any technical changes.

- [ ] [feat: change days to d](https://github.com/strangelove-ventures/sommelier/commit/4366f36cbfb592e286f66b9e71fbdd333c882194)
- [ ] [feat: create lp token icon](https://github.com/strangelove-ventures/sommelier/commit/54498be3d6830d89d1a7bc39d0791c35a4e61ed0)
- [ ] [feat: update bond modal view](https://github.com/strangelove-ventures/sommelier/commit/59f0ca51c3767403f6f9929fb3a70492b6c16649)

## Screenshots (if appropriate):
image

### CODE REVIEW

File: src/components/_forms/BondForm/index.tsx [hunk 1]
```diff
@@ -152,39 +153,75 @@ export const BondForm: VFC<BondFormProps> = ({ onClose }) => {
...
+            borderRadius={16}
+            appearance="none"
+            textAlign="start"
+            css={{
+              "span:first-of-type": {
+                display: "flex",
+                justifyContent: "space-between",
+                alignItems: "center",
+              },
+            }}
```
Comment (xanderjl) [inline src/components/_forms/BondForm/index.tsx:172]: I'm pretty sure you can move these styles into the `_first` prop of the `<Text as="span" ... />` component on line 206!

**Review by xanderjl — APPROVED** [review]
LGTM! Just one minor suggestion re: not needing the `css` prop on the parent form div. Totally up to you if you want to try it. Not a blocker. This is good to go 💪

File: src/components/_forms/BondForm/index.tsx (hunk 1 above)
Comment (joshuanatanielnm) [reply src/components/_forms/BondForm/index.tsx]: Thanks Xan, I'll implement this

**Comment by codingki** [comment]
Hey @joshuanatanielnm I think you forgot to change these days format https://github.com/strangelove-ventures/sommelier/pull/754/files#diff-6dbcfe6a891344cb90946eeb00ea85808aa8e31fcbeb62cf9a46dfafe55d4d50R194-R232 or is the changes expected for only aave cellars?

**Comment by codingki** [comment]
Sorry this makes my eye itchy 😂 really small UI missmatch I notice 7D 14D 21D is have a little bit bigger font size,
figma: 
CleanShot 2022-12-08 at 07 06 56

**Comment by codingki** [comment]
oh one more thing "LP Tokens" should be the token name you can get it from:
```
const { lpTokenInfo } =  useUserBalances(cellarConfig)
lpTokenInfo.data.name
```

**Pull request merged**